src/components/views/elements/AccessibleTooltipButton.tsx

```diff
@@ -80,6 +80,8 @@ export default class AccessibleTooltipButton extends React.PureComponent<IToolti
                 {...props}
                 onMouseOver={this.onMouseOver}
                 onMouseLeave={this.onMouseLeave}
+                onFocus={this.onMouseOver}
+                onBlur={this.onMouseLeave}
```

Comment:
If you're re-using the handler for multiple events, can you either rename the handler or add separate handler functions calling a common one, otherwise I think this can get confusing.